Create a map indicating locations of Gizra's team.

nedSaf opened the pull request:
#606

nader77 commented:
Update: Added the map:
![the_team](https://user-images.githubusercontent.com/7369740/28968094-0799f774-7927-11e7-8f3a-d19e8807d238.jpg)


Next: Handling ports to send markers to JS.

amitaibu commented on team_app/src/GizraTeam.elm:
```diff
@@ -546,3 +492,88 @@ people =
+
+mapMarkers : List Marker
+mapMarkers =
+    [ { id = "amitaibu"
```
Happy to see you Elming :) 

If we'll make it team `EveryDictList` we could convert the names to be a proper types

```elm
type Team 
   =  Amitaibu
   | Bricel
  | OritiMG
 -- ...
```

amitaibu replied:
(only if it makes sense, maybe it would make more sense to move the markers info under the team's DictList)

nader77 replied:
👍  I'll check it out.

nader77 replied:
@amitaibu Ports take only certain types like `Array, List, String ...` and some default ones looks like I'll have to use the `JSON` decode/encode to pass the custom union types we have inside `People`, do you know something about that?

nader77 commented:
Update: Map toggle functionality. 
![aug-05-2017 01-22-15](https://user-images.githubusercontent.com/7369740/28989223-aa8c3c9e-797c-11e7-9eb8-8439bda68a53.gif)


The map with locations:
![the_team](https://user-images.githubusercontent.com/7369740/28989238-bb1c2272-797c-11e7-831d-7ffee934c0e4.jpg)

nader77 commented:
Update: Added locations of existing team members.

BTW @RachelBaram @OritGeron there are some team members missing, surprisingly @noamoss 😕 

Next: 
* Check if we can change types or put location data inside the the person's info (original approach).
* Prefer to get this data inside the user so I can use the memeber's picture as the "Marker".
* Add a popup with some info about the member.
* Chnage the toggle map button to: `Grid view` and `Map view` buttons with icons.

RachelBaram commented:
@nader77 - we agreed to allow Noam not to put up his picture on our website. We have two options 1. not include him at all 2. include him with a generic avatar.  @noamoss what do you feel the most comfortable with?

noamoss commented:
@RachelBaram both are fine by me. If there is no problem adding a generic avatar I would be glad to share my info.

( @nader77 thanks for noticing :) )

nader77 commented:
@RachelBaram @noamoss Generic avatar it is for all the employees until we get pictures.

nedSaf commented:
Added a link to toggle between grid and map views:
![the_team](https://user-images.githubusercontent.com/7369740/29135452-d6a8a30c-7d42-11e7-8d05-565bd076d3d6.jpg)

nedSaf commented:
The flow:

![aug-09-2017 20-44-16](https://user-images.githubusercontent.com/7369740/29135671-97be32be-7d43-11e7-9b6b-a50271401e62.gif)

nedSaf commented:
![the_team](https://user-images.githubusercontent.com/7369740/29135717-bd8a72fa-7d43-11e7-8dd9-cfb59033b656.jpg)

nedSaf commented:
@RachelBaram @noamoss @OritGeron Hope you like it, any more ideas for improvements?

Next: mark which view the user is now seeing.

RachelBaram commented:
@nedSaf Cool stuff. How do we deal with the overlap of all the Israel employees?

RachelBaram commented:
@IshaDakota  Just making sure you are in the loop.

nedSaf commented:
@RachelBaram Well, we can zooming buttons. i.e. `Israel` and this button will zoom on Israel's area.

I will look into pointers like an arrow or something and that would make it possible to spread the images, I just don't think it's possible, I'll just have a look around.

nedSaf commented:
A nice solution: https://github.com/Leaflet/Leaflet.markercluster

nedSaf commented:
@RachelBaram 
![sep-07-2017 17-47-28](https://user-images.githubusercontent.com/7369740/30169249-
7f4ac77a-93f4-11e7-92fe-45770df81563.gif)

nedSaf commented:
This is how it looks now, what do we think?

RachelBaram commented:
Nice!

OritGeron commented:
@nedSaf Bravo!

RachelBaram commented:
@IshaDakota Can you review and merge please.

nedSaf commented:
@RachelBaram I need to review it myself first and then give it to @amitaibu to review the elm part then we can give it to @IshaDakota, I need to update the team as well, remove one profile and add Luis, I don't think I have any information about him.

nedSaf commented:
I will finish it today after work :)

nedSaf commented:
Updated toggle links:
![sep-10-2017 17-48-16](https://user-images.githubusercontent.com/7369740/30249994-19f4ec12-9650-11e7-8156-875fae3c048f.gif)

nedSaf commented:
@amitaibu @IshaDakota Ready for review.

IshaDakota commented on src/_layouts/default.html:
```diff
@@ -94,6 +99,11 @@
 </div>
   <script src="/bower_components/anchor-js/anchor.min.js"></script>
   <script src="/assets/javascript/script.js"></script>
+  {% if page.title == 'The Team' %}
```
nitpick: It's less likely that we would change the URL than the page tltle.

IshaDakota commented on src/assets/scss/_team.scss:
```diff
@@ -27,9 +50,18 @@
+
+  #map {
+    width: 100%;
+    height: 700px;
```
for me, the map is a little tall. On a 13-inch screen the whole map isn't visible and and I don't see the zoom pagers. Since most of us don't live at the poles, it could probably tolerate being more landscape.

OritGeron commented:
![the_team](https://user-images.githubusercontent.com/7068965/30260848-6b62cf10-96d2-11e7-92c5-5fd8d7b2acf7.jpg)

@nedSaf

nedSaf commented:
Fixed @IshaDakota comments and removed all people without a picture:
![the_team](https://user-images.githubusercontent.com/7369740/30284433-988a7a78-9723-11e7-805c-35d4050ac597.jpg)

nedSaf commented:
@IshaDakota @amitaibu This is ready.

nedSaf commented:
@amitaibu @IshaDakota Do we want to get this in or we're not sure about this, in that case, I'll create a PR for updating the team members.

amitaibu commented:
Sorry, I missed that it needs review.

I'm ok with getting this in. @IshaDakota can you deploy please.
